modular_skyrat/modules/mapping/code/lavaland_ruin_code.dm

```diff
@@ -17,6 +17,17 @@
 	you_are_text = "You are a syndicate science technician, employed in a top secret research facility developing biological weapons."
 	outfit = /datum/outfit/lavaland_syndicate/ice
 
+/obj/effect/mob_spawn/ghost_role/human/lost_veterinarian
+	name = "Lost Veterinarian"
+	you_are_text = "You're a doctor waking up in a land unfamiliar to you, searing heat surrounds your former veternarian clinic."
+	flavour_text = "Unfortunately you have no idea how you got here, if your customers are still alive or even what year it is."
+	important_text = "This is a make your own journey ghost-role, it is allowed to go to the station. You aren't mean to attack anyone directly, you're just a vet after all."
```

Comment:
```suggestion
	important_text = "This is a make your own journey ghost-role, it is allowed to go to the station. You aren't meant to attack anyone directly, you're just a vet after all."
```
the second sentence feels entirely unneeded

Reply:
think so? Idea was just to clear the baseline up front - it do come off a lil aggressive